AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -16,30 +16,37 @@ internal protocol CardViewControllerProtocol {
 }
 
 internal class CardViewController: FormViewController {
-
+    
     private let configuration: CardComponent.Configuration
-
     private let shopperInformation: PrefilledShopperInformation?
-
     private let supportedCardTypes: [CardType]
-
     private let formStyle: FormComponentStyle
-    
     private var issuingCountryCode: String?
-    
     private let payment: Payment?
-    
     private let initialCountryCode: String
-    
     private let scope: String
-    
     private let cardLogos: [FormCardLogosItem.CardTypeLogo]
+    private lazy var cardScannerController: CardScannerControlling = {
+        let controller = CardScannerController(presenter: self)
+        controller.onScanComplete = { [weak self] result in
+            switch result {
```

Comment:
maybe better to extract this into a function

Reply:
Was on my mind and forgot in the end, good one.